cmd/utils/flags.go

```diff
@@ -1860,6 +1866,9 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	if ctx.IsSet(RollupSequencerHTTPFlag.Name) && !ctx.IsSet(MiningEnabledFlag.Name) {
 		cfg.RollupSequencerHTTP = ctx.String(RollupSequencerHTTPFlag.Name)
 	}
+	if ctx.IsSet(RollupDisableTxPoolFlag.Name) {
+		cfg.RollupDisableTxPoolGossip = true
+	}
```

Comment:
Reading this more closely, it should be:
```
cfg.RollupDisableTxPoolGossip = ctx.Bool(RollupDisableTxPoolFlag.Name)
```
Since the user can pass `--rollup.disabletxpool=false` and the `IsSet` method would find the flag was declared by the user, and set it to true.

Reply:
Left the `IsSet` but used `ctx.Bool` on the flag name. Similar to existing bool flags. Also renamed flag var for consistency.